Duplicati/UnitTest/UtilityTests.cs

```diff
@@ -495,6 +495,34 @@ public static void ThrottledStreamWrite()
                 }
             }
         }
+
+        [Test]
+        [Category("Utility")]
+        public static void RetryDelayNoExponentialBackoffTest()
+        {
+            // test boundary values
+            TimeSpan baseDelay = TimeSpan.FromSeconds(1);
+
+            int[] testValues = { 1, 2, 63, 64, int.MaxValue };
+            double[] expect =  { 1, 1,  1,  1,            1 };
+
+            for (int i = 0; i < testValues.Length; i++)
+                Assert.AreEqual(TimeSpan.FromSeconds(expect[i]), Utility.GetRetryDelay(baseDelay, testValues[i], false));
+        }
+
+        [Test]
+        [Category("Utility")]
+        public static void RetryDelayExponentialBackoffTest()

```

Comment:
Minor issue, but I think we can omit `Test` from the end of the method name.

Reply:
Done.